custom_components/tuya_local/devices/reiga_52_fan_light.yaml

```diff
@@ -0,0 +1,77 @@
+name: Reiga Ceiling Fan
```

Comment:
HA naming convention: Only capitalise the first word of a device or entity name, unless it is an abbreviation or proper name that needs capitalising even in the middle of sentences.  I also prefer to leave brand names off the top level name if there is specific product info listed, as there are often other brands sold in different markets, or different retailers in the same market that are the same product rebranded.